app/models/pacbio/well.rb

```diff
@@ -111,7 +77,7 @@ def summary
 
     # A collection of all the used_aliquots for given libraries and pools in a well
     def base_used_aliquots
-      used_aliquots.collect(&:source).collect(&:used_aliquots).flatten
+      used_aliquots.reject(&:marked_for_destruction?).collect(&:source).collect(&:used_aliquots).flatten # rubocop:disable Layout/LineLength
```

Comment:
you can have this span multiple lines and remove the rubocop line layout. If you remove the comment and run `bundle exec rubocop -A` that will attempt to autofix errors like this.

Reply:
I am still getting that error even after running bundle exec rubocop -A